mobile/library/cc/BUILD

```diff
@@ -12,7 +12,7 @@ envoy_cc_library(
     hdrs = [
         "engine_builder.h",
     ],
-    copts = select({
+    copts = envoy_mobile_copts("@envoy") + select({
```

Comment:
Why is this needed? We should already be getting these from `envoy_cc_library`, no?

Reply:
doesn't need it. ended up scrapping this entire buld file and plumbing a different way to catch tests not using the config template.  That said it'd still a draft so feel free to ignore until I've gotten to clean ups